src/apps/settings/src/app/views/Robots/Robots.js

```diff
@@ -154,15 +164,30 @@ export default connect((state) => {
         />
 
         <div className={styles.Row}>
-          <FieldTypeBinary
-            name="settings[general][robots_on]"
-            label={robotOn.keyFriendly}
-            tooltip={robotOn.tips}
-            value={Boolean(robotOn.value)}
-            offValue="No"
-            onValue="Yes"
-            onChange={handleRobotsOn}
-          />
+          <FormLabel sx={{ color: "primary.dark" }}>
+            <Stack
+              direction="row"
+              alignItems="center"
+              sx={{
+                my: 1,
+              }}
+            >
+              <Tooltip title={robotOn.tips} arrow placement="top-start">
+                <InfoIcon fontSize="small" sx={{ mr: 1 }} />
+              </Tooltip>
+              {robotOn.keyFriendly}
+            </Stack>
+          </FormLabel>
+          <ToggleButtonGroup
+            color="secondary"
+            size="small"
+            value={robotOn.value}
+            exclusive
+            onChange={(evt, val) => handleToggle(val)}
+          >
+            <ToggleButton value={0}>No </ToggleButton>
```

Comment:
These values are numeric and the other two are strings. Is this intended? I would expect them all to be the same type since they were all coming from the FieldTypeBinary component

Reply:
Okay converted to string  and cleaned up state conversion to numbers and then back to strings. Now strings are being passed all the way through as the API expects strings